content/enterprise_influxdb/v1.8/about-the-project/release-notes-changelog.md

```diff
@@ -9,11 +9,23 @@ menu:
     parent: About the project
 ---
 
+## v1.8.5 [2020-04-20]
+
+The InfluxDB Enterprise 1.8.5 release builds on the InfluxDB OSS 1.8.5 release.
+For details on changes incorporated from the InfluxDB OSS release, see
+[InfluxDB OSS release notes](/influxdb/v1.8/about_the_project/releasenotes-changelog/#v185-2021-04-20).
+
+### Bug fixes
+
+- Resolve TSM backup "snapshot in progress" error.
+- Add labels to the values returned in SHOW SHARDS output to clarify the node ID and TCP address.
+- Always forward repairs to the next data node (even if the current data node does not have to take action for the repair).
```

Comment:
There are three other items in the Plutonium changelog, are these minor enough that they don't need to be in this document?

Reply:
yes... we need to add: Meta queries now respect QueryTimeout values, SHOW DATABASES now only shows databases that the user has either read or write access to, `influxd_ctl entropy` now shows shard expiry times which are consistent with `influxd_ctl show-shards`

Reply:
the meta query stuff is already in OSS...but I think there were some specific fixes that were needed for Enterprise.  So, I don't think we need to repeat that item.

Reply:
Thank you, I was looking at the 1.8.5 list in the master branch. Updated!